src/components/NavigationBar/NavigationBar.tsx

```diff
@@ -35,6 +39,18 @@ const NavigationBar = decorate<OwnProps>(
             >
               Administration
             </Typography>
+            <Button>
```

Comment:
from specs: https://material-ui-next.com/demos/app-bar/

I think we should add "color="contrast""

Reply:
color="inherit" worked but only for flat buttons. need to investigate more